input/pagecontent/smart-web-messaging.md

```diff
@@ -455,6 +467,59 @@ targetWindow.postMessage({
 }, targetOrigin);
 ```
 
+If an app were to request the entire contents of the scratchpad by submitting a
+`scratchpad.read` without specifying a `location` parameter, the EHR might respond with
+a bundle containing multiple resources.
```

Comment:
The value below is not a FHIR Bundle; I was expecting to see a Bundle, from the proposed resolution on FHIR-33013

Reply:
Yeah, I struggled with how to show this.  I wanted to illustrate that the scratchpad could represent the objects however it wished (not necessarily as a bundle), but I think you're right that it doesn't need to be there.  The important part of this example is showing a collection bundle returned from a scratchpad.read operation with no location set.